mentoring/templates/html/ratingblock.html

```diff
@@ -6,38 +6,48 @@
   <div class="choices-list">
     <div class="choice">
       <div class="choice-result fa icon-2x"></div>
-      <label><input class="choice-selector" type="radio" name="{{ self.name }}" value="1"{% if self.student_choice == '1' %} checked{% endif %} />1</label>
-      <span class="low"> - {{ self.low }}</span>
-      <div class="choice-tips"></div>
+      <label class="choice-label"><input class="choice-selector" type="radio" name="{{ self.name }}" value="1"{% if self.student_choice == '1' %} checked{% endif %} />1 <span class="low">- {{ self.low }}</span></label>
```

Comment:
Nitpick -- these lines are getting really long. They would be more readable if you broke them up, for example:
```html
<label class="choice-label">
  <input class="choice-selector" type="radio" name="{{ self.name }}" value="1"
    {% if self.student_choice == '1' %} checked{% endif %} />
  1 <span class="low">- {{ self.low }}</span>
</label>
```